chain/indexer/CHANGELOG.md

```diff
@@ -1,56 +1,68 @@
 # Changelog
 
+## 1.25.x (UNRELEASED)
+
+- `state_changes` is moved to `IndexerShard` struct and represent only
+  the state changes happened on the specific shard
+- All the NEAR Indexer Framework types was extracted to a separate crate `indexer-primitives`
+
+## Breaking changes
+
+The field `state_changes` is moved from the root of `StreamerMessage`
+to the `IndexerShard.state_changes` and now contains only changes related
+to the specific shard.
+
 ## 0.10.1
 
-* (mainnet only) Add additional handler to inject restored receipts to the block #47317863. See [PR 4248](https://github.com/near/nearcore/pull/4248) for reference
+- (mainnet only) Add additional handler to inject restored receipts to the block #47317863. See [PR 4248](https://github.com/near/nearcore/pull/4248) for reference
```

Comment:
I suggest we rollback the changes regarding the use of `*` vs `-` and other formatting changes unless there is some contribution guide that defined this rule.

Reply:
It was an imposter - VSCode :( I didn't notice it. Thanks!